webapp/src/main/webapp/resources/template.soy

```diff
@@ -382,4 +382,39 @@
   </table>
   <p>Please fix the stops positions to make their distance further than the trip buffer meters!</p>
   <br><br>
+{/template} 
+
+/**
+ * Renders a basic explanation of the stop too far from trip shape notice.
+ * @param notices : List<[stopId : string, stopSequence : number, tripId : string, shapeId : string, tripBufferMeters : number]>
+ */
+{template .stopTooFarFromTripShape}
+{let $numNotices: length($notices)/}
+  <p class="error">Error - Stop too far from trip shape!</p>
+  <p>Description: Stop is too far away from the trip shape.</p>
+  <p><b>{$numNotices}</b> Stop(s) too far from trip shape found in:</p>
```

Comment:
This doesn't make sense grammatically. I also think there is too much repetition, so it is sufficient to say 
<p><b>{$numNotices}</b> found:</p>

Reply:
Yes. Have adjusted all templates here.